tarball-default.nix

```diff
@@ -1,11 +1,11 @@
-let
+rec {
   alamgu = import (fetchTarball "https://github.com/alamgu/alamgu/archive/develop.tar.gz") {};
```

Comment:
This PR is going to develop so I didn't touch that, but yes `main`'s should definitely be `main`, and frankly yeah this could probably get away with being `main` too.